tools/fiptool/fiptool.c

```diff
@@ -646,7 +646,10 @@ static unsigned long get_image_align(char *arg)
 	unsigned long align;
 
 	errno = 0;
-	align = strtoul(arg, &endptr, 10);
+	if (strlen(arg) > 2 && strncmp(arg, "0x", 2) == 0)
+		align = strtoul(arg + 2, &endptr, 16);
```

Comment:
I would rather see strtoul() use base 0 (it will detect if it is hex, dec or octal automatically).  Avoids having to detect the base by scanning for 0x etc.